DATA/production/qc-sync/phs.json

```diff
@@ -0,0 +1,69 @@
+{
+  "qc": {
+    "config": {
+      "database": {
+        "implementation": "CCDB",
+        "host": "ali-qcdb.cern.ch:8083",
+        "username": "not_applicable",
+        "password": "not_applicable",
+        "name": "not_applicable"
+      },
+      "Activity": {
+        "number": "42",
+        "type": "2"
+      },
+      "monitoring": {
+        "url": "influxdb-unix:///tmp/telegraf.sock"
+      },
+      "consul": {
+        "url": "http://ali-consul.cern.ch:8500"
+      },
+      "conditionDB": {
+        "url": "http://localhost:8084"
+      }
+    },
+    "tasks": {
+      "ClusterTask": {
+        "active": "true",
+        "className": "o2::quality_control_modules::phos::ClusterQcTask",
+        "moduleName": "QcPHOS",
+        "detectorName": "PHS",
+        "cycleDurationSeconds": "100",
+        "maxNumberCycles": "-1",
+        "dataSource": {
+          "type": "dataSamplingPolicy",
+          "name": "phos-clu"
+        },
+        "taskParameters": {
+          "": ""
+        },
+        "location": "local",
+        "localMachines": [
+          "localhost"
+        ],
+        "remoteMachine": "alio2-cr1-qme06.cern.ch",
+        "remotePort": "47758",
+        "mergingMode": "delta",
+        "localControl": "odc"
+      }
+    }
+  },
+  "dataSamplingPolicies": [
+    {
+      "id": "phos-clu",
+      "active": "true",
+      "machines": [
+        "epn"
+      ],
```

Comment:
Are you sure this works on EPN? in the sampling policy you use the "epn" tag, but in `ClusterTask` it is `localhost`. A local QC workflow can be executed with one `--host` parameter, so either the policy (if `epn`) or the task (if `localhost`) will be spawned, never both.
As far as I remember, EPN QC still uses `--host localhost`. We should probably organize a campaign to move to `epn` which is clearer.

Reply:
Hi @knopers8 
I have 2 questions regarding that. 
1) What is "localMachines[ ]" ? Is it place where objects for merging are produced?
2) What is "machines" in dataSamplingPolicies?

Reply:
And one more question: on epns we use "epn" machine to indicate local part of task, or "localhost"? I'm abit confused. Is it just label for machine or real address?

Reply:
I will refer you to the doc:
https://github.com/AliceO2Group/QualityControl/blob/master/doc/Advanced.md#multi-node-setups
https://github.com/AliceO2Group/QualityControl/blob/master/doc/Advanced.md#configuration-files-details
if it is still unclear, let me know.

regarding the second question, I believe that `localhost` is still used to label tasks which should run on EPNs. It is probably a remnant of some test setup that propagated to the production.

Reply:
hi @knopers8 
yes, one more question. If we put localMachines[ "epn" ] then on every epn node same workflow ...| o2-qc --local --host epn ... is spawned, right? Sorry, I don't understand it. What word "enp" stands for? Is it just label or real machine address?

Reply:
It is just a label.
This basically allows you to have one QC config file for both FLPs, EPNs and QC servers (with mergers). Then you can e.g. execute `o2-qc --local --host epn` to spawn only tasks which have the "epn" label, exactly as you say.
You can also have several `localMachines` and then a task will be spawned if `--host` matches any of them. `machines` in data sampling follow the same principle, but if the list is empty, it will be always spawned (for backwards compatibility).

Reply:
thanks for explanation. Still I have question:
in the qc-workflow.sh qc is added by following command:
`  add_W o2-qc "--config json://$QC_JSON_FROM_OUTSIDE ${QC_CONFIG_PARAM:---local --host ${QC_HOST:-localhost}}" "" 0
`
if I understood correctly, QC_HOST is not defined anywhere so all tasks at epn are now submitted with 'localhost' label so if we want json files to work then all the local parts must be labeled as 'localhost', right?

Reply:
Yes, that is correct.

We will have to organize a campaign to move to "epn". For now, please use "localhost".

Reply:
Ah, ok! Now I undestood completely.

Reply:
hi @knopers8 
can you please look again into qc-sync/phs.json? Thanks!

Reply:
also I din't include phs.json to qc-workflow.sh yet because corresponding adjusted dpl-workflow.sh must be deployed to P2 first in order it to be tested.

Reply:
The file looks correct to me.

Is it ready to merged then?

Reply:
Hi @knopers8 
I will have chance to test phs.json at epn as soon as https://github.com/AliceO2Group/AliceO2/pull/8224 appears at epn. 
As for cpv, it is tested fine. So I propose to merge it for now.